vnext/Microsoft.ReactNative.Cxx/ReactDispatcher.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#ifndef MICROSOFT_REACTNATIVE_REACTDISPATCHER
+#define MICROSOFT_REACTNATIVE_REACTDISPATCHER
+
+#include <winrt/Microsoft.ReactNative.h>
+#include "ReactHandleHelper.h"
+
+namespace winrt::Microsoft::ReactNative {
+
+// Represents a dispatcher queue to invoke work item asynchronously.
+// It wraps up the IReactDispatcher and adds convenience methods for
+// working with C++ types.
+struct ReactDispatcher {
+  ReactDispatcher(std::nullptr_t = nullptr) noexcept {}
+
+  explicit ReactDispatcher(IReactDispatcher const &handle) noexcept : m_handle{handle} {}
```

Comment:
> & [](start = 49, length = 2)

(nit) Since we are doing assignment anyways, wouldn't && be better here?

Reply:
It is the C++/WinRT thing. It always gives us const &.

---
In reply to: [427624661](https://github.com/microsoft/react-native-windows/pull/4953#discussion_r427624661) [](ancestors = 427624661)